ocs_ci/ocs/resources/storage_cluster.py

```
@@ -155,6 +157,149 @@ def verify_osd_tree_schema(ct_pod, deviceset_pvcs):
+
+    remove_storageclass_from_storageconsumer(sc_name)
+    patch_storageclass_ocs_external_label(sc_name)
+
+    if sc_ocp.is_exist(resource_name=sc_name):
+        log.info(f"Deleting StorageClass {sc_name}")
+        sc_ocp.delete(resource_name=sc_name)
+
+    end_time = time.time() + timeout
+    while time.time() < end_time:
+        if not sc_ocp.is_exist(resource_name=sc_name):
+            log.info(f"StorageClass {sc_name} confirmed deleted")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)